dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -125,4 +127,16 @@ public void runWESCommand(String jsonInputFilePath, File localPrimaryDescriptorF
             }
         }
     }
+
+    /**
+     * help text output
+     */
+    private void wesCommandSuggestions() {
+        String wesEntryType = abstractEntryClient.getEntryType().toString().toLowerCase();
+        out("Useful commands for working with WES runs:");
+        out("");
+        out("To get status for a run: dockstore " + wesEntryType + " wes status --id <run id> [--verbose][--wes-url <WES URL>][--wes-auth <auth>]");
```

Comment:
My idea was that you put the actual id in there so you can just copy and paste it directly, rather than copying that, then copying the id, then combine them together by editing a line... Something like:
```
To get the status of this run: dockstore wes status --id <the id from line  117/118>. 
```

You may have to word it more carefully to handle extra parameters, but presumably a user is going to want to check on the status, so just give them the exact command to run, if possible.

Reply:
Added the run id to the message